velox/type/DecimalUtil.h

```diff
@@ -250,6 +250,21 @@ class DecimalUtil {
     return (unsignedSum >> 127);
   }
 
+  /// Adds two int128_t inputs, calculate the sum, and return the overflow.
+  /// It can be used to track the number of overflow when adding a batch of
+  /// input numbers. It takes lhs and rhs as input, and stores their sum in
+  /// result. overflow == 1 indicates upward overflow. overflow == -1
+  /// indicates downward overflow. overflow == 0 indicates no overflow.
+  /// When only one input is negative, adding them will never result in
+  /// overflow, so we can directly add them. When both inputs are positive
+  /// or negative, adding them may result in overflow. We can first convert
```

Comment:
nit: "We can ..." - make it clear that this is what this function does, not just what it could do.

`To add numbers that may overflow, first convert both numbers to unsigned 129-bit number (uint128_t) and perform the addition. The highest bits in the result indicates overflow.`